### PR TITLE
feat: add result and result.schema and parameters.schema to method docs

### PR DESCRIPTION
**Pull request by shanejonas**
image

### CODE REVIEW

File: templates/method.template.md.js [hunk 1]
```diff
@@ -1,11 +1,36 @@
+
 module.exports = require('lodash').template(`
+
+<% const makeSchema = (property, value) => {%>
+| <%= property %> | <%= value %> | <% } %>
```
Comment (shanejonas) [inline templates/method.template.md.js]: oh this is extra garbage

File: templates/method.template.md.js [hunk 2]
```diff
@@ -1,11 +1,36 @@
+
 module.exports = require('lodash').template(`
+
+<% const makeSchema = (property, value) => {%>
```
Comment (shanejonas) [inline templates/method.template.md.js]:
```suggestion

```

File: templates/method.template.md.js (hunk 1 above)
Comment (shanejonas) [inline templates/method.template.md.js]:
```suggestion

```

File: src/index.js [hunk 3]
```diff
@@ -8,7 +10,7 @@ module.exports = ({info, methods, servers }) => {
   const infoDocs = templates.info({info});
   const serverDocs = templates.server({servers});
   const methodDocs = methods.map((method) => {
-    return templates.method({method});
+    return templates.method({method, schemaToMarkdown});
```
Comment (BelfordZ) [inline src/index.js]: since the file is js now, you dont need tp inject

File: templates/method.template.md.js [hunk 4]
```diff
@@ -1,11 +1,34 @@
+
```
Comment (BelfordZ) [inline templates/method.template.md.js]:
```suggestion
const schemaToMarkdown = require('json-schema-to-markdown-table');
```

File: src/index.js (hunk 3 above)
Comment (BelfordZ) [inline src/index.js]:
```suggestion
    return templates.method({method});
```

File: src/index.js [hunk 5]
```diff
@@ -1,3 +1,5 @@
+const schemaToMarkdown = require('json-schema-to-markdown-table');
```
Comment (BelfordZ) [inline src/index.js]:
```suggestion
```

**Pull request merged**